### PR TITLE
Bring workshop toolbox in line with nested containers

### PR DESCRIPTION
**Pull request by Mom-Bun**
#### Summary

SUMMARY: Bugfixes "Give workshop toolbox pockets"

#### Purpose of change

Nested containers hit awhile ago, and currently workshop toolboxes are still craftable and act like the magical multi-tool, instead of a box like the regular toolbox.

#### Describe the solution

Changes the workshop toolbox to have pockets, and acting like a bigger toolbox that can fit more tools, buffing up the skill requirements and materials required to craft it as well.

I also changed the workshop item group to spawn with this new toolbox, instead of the current one.

#### Describe alternatives you've considered

Removing it outright, as having a magical multi-tool isn't right.

#### Testing

Booting up the game, no errors shoot out, and the new toolbox acts correctly.

### CODE REVIEW

**Comment by Fris0uman** [comment]
This will turn the current workshop box into an empty box and lose all the tools it had inside, maybe you should inside make a new item `toolbox_workshop_empty` that is the empty box and add a disassemble recipe to the legacy toolbox_workshop to give back the empty box and the tools. And then obsolete the toolbox_workshop and replace it with the empty one in the item groups

**Comment by Mom-Bun** [comment]
> This will turn the current workshop box into an empty box and lose all the tools it had inside, maybe you should instead make a new item `toolbox_workshop_empty` that is the empty box and add a disassemble recipe to the legacy toolbox_workshop to give back the empty box and the tools. And then obsolete the toolbox_workshop and replace it with the empty one in the item groups

To note, the workshop toolbox didn't spawn anywhere, as it was replaced by the toolbox_empty, so afaik unless some players crafted it, but I can do that

**Comment by Fris0uman** [comment]
> the workshop toolbox didn't spawn anywhere, 

It would be cool if you could make some workshop tool box tem groups for it and them spawn then

**Comment by Mom-Bun** [comment]
> > the workshop toolbox didn't spawn anywhere,
> 
> It would be cool if you could make some workshop tool box item groups for it and them spawn then

Ah, I added them to the workshop_toolbox spawngroup, that prior was using the regular toolbox_empty, which should now use workshop_toolbox_empty

**Comment by Mom-Bun** [comment]
Switching this to a draft, as workshop toolbox would be better as a vehicle than an item.

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -1419,29 +1419,16 @@
...
     "type": "recipe",
     "activity_level": "fake",
-    "result": "toolbox_workshop",
+    "result": "toolbox_workshop_empty",
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
     "skill_used": "fabrication",
-    "skills_required": [ "mechanics", 3 ],
-    "difficulty": 4,
+    "skills_required": [ "mechanics", 5 ],
```
Comment (Fris0uman) [inline data/json/recipes/recipe_others.json:1426]: Why the increase in difficulty it's just a box, no?

**Pull request merged**